generators/_init-web/templates/plain/.eslintrc.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+  root: true,
+  extends: [
+    "airbnb",
+    "airbnb-typescript",
+    "airbnb/hooks",
+    "eslint:recommended",
+    "plugin:import/recommended",
+    "plugin:react/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:jest/recommended",
+    "plugin:prettier/recommended"
+  ],
+  plugins: ["react", "@typescript-eslint", "jest"],
+  ignorePatterns: ["*.js"],
+  env: {
+    browser: true,
+    es6: true,
+    jest: true
+  },
+  globals: {
+    Atomics: "readonly",
+    SharedArrayBuffer: "readonly",
+  },
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    // Make sure eslint and VS Code use the same path for the tsconfig.json:
+    // https://github.com/typescript-eslint/typescript-eslint/issues/251
+    tsconfigRootDir: __dirname,
+    project: "./tsconfig.eslint.json"
+  },
+  rules: {
+    // Disables jsx-a11y https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/no-webpack-loader-syntax.md
+    ...Object.keys(require('eslint-plugin-jsx-a11y').rules).reduce((acc, rule) => { acc[`jsx-a11y/${rule}`] = 'off'; return acc }, {}),
+    "class-methods-use-this":"off",
+    "linebreak-style": "off",
+    "no-continue": "off",
+    "no-multi-assign": "warn",
+    "no-nested-ternary": "off",
+    "no-param-reassign": ["error", { "props": false }],
+    "no-return-assign": "warn", 
+    "no-restricted-syntax": "off",
+    "no-plusplus": "off",
+    "no-prototype-builtins": "warn",
+    "no-minusminus": "off",
+    "no-underscore-dangle": "off",
+    "max-classes-per-file": "off",
+    "no-param-reassign": "warn",
```

Comment:
```suggestion
    "no-param-reassign": ["warn", { props: true, ignorePropertyModificationsFor: ['state'] }],
```

If we keep this line, please change. The lint configuration is also [recommended by the Redux Toolkit Team](https://redux-toolkit.js.org/usage/immer-reducers#linting-state-mutations). Otherwise we get a lot of non-fixable warnings with the Redux `state` variable. 

![grafik](https://user-images.githubusercontent.com/5851088/159041761-74138637-b1fc-455c-a142-1d21778530b3.png)

Reply:
@thinkh I agree that we should allow `immer` style updates via `"no-param-reassign": ["warn", { props: true, ignorePropertyModificationsFor: ['state'] }],`. @dvdanielrehberger Can you remove the `no-param-reassign` above and use this instead? 
@thinkh What we should think of is a common `datavisyn-scripts` library (similar to `react-scripts`), were we place such configurations. That would allow us to put all dependencies (eslint, prettier, typescript?, ...) there, and simply `extend` it in the configuration files.

Reply:
@puehringer Both sounds good to me. Having a common script package would make maintenance easier. Should we create an issue for that?

@dvdanielrehberger I will approve this PR once you addressed the changes.

Reply:
@thinkh Done: https://github.com/datavisyn/visyn2025/issues/131